src/test/data/JsonSerializableCatalogTest/invalidBookCatalog.json

```diff
@@ -1,8 +1,8 @@
 {
   "books": [ {
-    "title": "Hans Muster",
+    "title": "&Hans Muster",
     "serialNumber": "invalid9482424",
-    "author": "invalid@email!3e",
+    "author": " invalid@email!3e",
```

Comment:
Since we are always trimming the user's input, I think this example for the author will never happen

Reply:
noted. Thanks!